mongo/integration/unified/matches.go

```diff
@@ -165,6 +184,39 @@ func verifyValuesMatchInner(ctx context.Context, expected, actual bson.RawValue)
 	return nil
 }
 
+func compareDocumentToString(ctx context.Context, expected, actual bson.RawValue) error {
```

Comment:
[opt] a comment above this function explaining that `expected` is a document to be compared to a string `actual`.